src/components/variable.jsx

```diff
@@ -48,51 +48,127 @@ const Variable = React.createClass({
   },
   renderVariableMetadata(variable) {
     const entityMessage = {
-      famille: "une famille",
-      foyer_fiscal: "un foyer fiscal",
-      individu: "un individu",
-      menage: "un ménage",
+      famille: "famille",
+      foyer_fiscal: "foyer fiscal",
+      individu: "individu",
+      menage: "ménage",
     }
     const definitionPeriodMessage = {
-      YEAR: "de l'année",
-      MONTH: "du mois",
-      ETERNITY: "de l'éternité (cette variable ne peut pas évoluer avec le temps)",
+      YEAR: <FormattedMessage id='aYear'/>,
+      MONTH: <FormattedMessage id='aMonth'/>,
+      ETERNITY: <FormattedMessage id='forEternity'/>
     }
+    const valueTypeMessage = {
+      Int: <FormattedMessage id='anInteger'/>,
+      Float: <FormattedMessage id='aFloat'/>,
+      Date: <FormattedMessage id='aDate'/>,
+    }
+    function formatDefaultValue(variable){
+    if (variable.valueType == "Date"){
+      return <FormattedDate value={variable.defaultValue} year='numeric' month='2-digit' day='2-digit'/>
+    }else if(variable.valueType == "Float"){
```

Comment:
For code readability, we add spaces around `{` and `}`:
```JS
} else if(variable.valueType == "Float") {
```

I'm adding this rule in the linter so that you get this feedback automatically when running the test :).